docs/source/ESPEasy/AboutUs.rst

```diff
@@ -1,2 +1,138 @@
 About Us
 ********
+The ESP Easy project is being handled by the core team of TD-er and Grovkillen.
+TD-er being the head of backend and Grovkillen being head of the frontend.
+We both dig down into each others areas of responsible on a daily basis though.
+
+Other than us we have around 100 volunteers that helping us maintain
+and improve the source code.
+
+History
+=======
+ESP Easy has gone through a lot of changes over the last years and here's some
+of the major milestones.
+
+Early days (Nodo Uno)
+---------------------
+It's a bit vague how it all started but Paul Tonkes (part of the Nodo team)
```

Comment:
When writing over the history, describing it as "vague" may not be the most profound way of describing it ;)